account_invoice_import_invoice2data/readme/CONFIGURE.rst

```diff
@@ -27,3 +29,76 @@ On the output, you will get first the text of the PDF, then some debug info on t
   * 'date': the date of the invoice
   * 'invoice_number'
   * 'date_due', if this information is available in the text of the PDF file.
+
+The invoice2data library is quite powerfull. It supports multiple input methods (pdftotext, ocrmypdf, tesseract ocr, google cloud vision).
```

Comment:
```suggestion
The invoice2data library is quite powerful. It supports multiple input methods (pdftotext, ocrmypdf, tesseract ocr, google cloud vision).
```
This seems more a collateral note. You can maybe add a section "Dependencies"

Reply:
Done, added dependencies section  :+1: